samples/snippets/csharp/tutorials/inheritance/is-a.cs

```diff
@@ -0,0 +1,61 @@
+﻿// <Snippet1>
+using System;
+
+public class Automobile
+{
+    public string _make;
```

Comment:
These should be private fields. (And I think using auto-implemented properties would be best).